playbooks/os/openshift-cluster/launch.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Launch instance(s)
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars_files:
+  - vars.yml
+  vars:
+    image_url: "{{ deployment_vars[deployment_type].image.url }}"
+    image_sha256: "{{ deployment_vars[deployment_type].image.sha256 }}"
+    image_name: "{{ deployment_vars[deployment_type].image.name }}"
+  tasks:
+  - fail:
+      msg: "Deployment type not supported for OpenStack provider yet"
+    when: deployment_type == 'online'
```

Comment:
I would like to use pre-configured RHEL 7.1 VM image.
I need to pre-configure them because openshift-ansible doesn’t do the registration yet.
But still, I need to use the `enterprise` deployment type, otherwise, the `origin` deployment type will setup yum repositories which are not suitable for RHEL.